proxy/src/project.rs

```diff
@@ -48,8 +48,28 @@ pub struct Project {
     /// Informs if the project is present in the Registry and under what top-level entity it can be
     /// found.
     pub registration: Option<Registration>,
-    /// High-level statistics about the project.
-    pub stats: Stats,
+    /// High-level statistics about the project
+    pub stats: coco::Stats,
+}
+
+/// Construct a Project from its metadata and stats
+impl Project {
+    /// Create a `Project` given a `librad` defined [`project::Project`] and the [`coco::Stats`]
+    /// for the repository.
+    pub fn from_project_stats<ST>(project: project::Project<ST>, stats: coco::Stats) -> Self
```

Comment:
Alternatively this could be a `From` impl for `(project::Project<ST>, coco::Stats)`.